prompb/io/prometheus/client/metrics.proto

```diff
@@ -147,4 +153,4 @@ message MetricFamily {
   string          help   = 2;
   MetricType      type   = 3;
   repeated Metric metric = 4 [(gogoproto.nullable) = false];
-}
+}
```

Comment:
Let's add empty line back (just convention)

Reply:
> Let's add empty line back (just convention)

I'm surprised that the linter allowed this 🤔